spec/services/foods_service_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe FoodsService do
+  before(:each) do
+    @fs = FoodsService.new
+  end
+
+  it "exists" do
+    expect(@fs).to be_an_instance_of FoodsService
+  end
+
+  it "can hit the /foods/search endpoint and provide a response", :vcr do
```

Comment:
Nice job using VCR - I like that it's only used on tests where it's absolutely necessary to do so.